Documentation/README.md

```diff
@@ -1,42 +1,43 @@
 # Ladybird Documentation
 
-Serenity development moves quickly, so some of these might be out of date. Please let us know if something here is wrong, or submit a PR with any additions or corrections! If you have any questions that are not answered here or in the [FAQ](FAQ.md), you are welcome to ask on [Discord](../README.md#get-in-touch-and-participate).
+Ladybird development moves quickly, so some of these might be out of date.
+Please let us know if something here is wrong, or submit a PR with any
+additions or corrections! If you have any questions that are not answered here
+or in the [FAQ](FAQ.md), you are welcome to ask on
+[Discord](../README.md#get-in-touch-and-participate).
 
 ## Building and Running
-* [Build Instructions](BuildInstructions.md)
-* [Advanced Build Instructions](AdvancedBuildInstructions.md)
-* [Troubleshooting](Troubleshooting.md)
-* [Running Tests](RunningTests.md)
-* [Setting Up Self-Hosted Runners](SelfHostedRunners.md)
-* [Profiling the Build](BuildProfilingInstructions.md)
-
-### OS-specific
-Make sure to read the basic [Build Instructions](BuildInstructionsLadybird.md) first.
-* [Building on Windows](BuildInstructionsWindows.md)
-* [Building on macOS](BuildInstructionsMacOS.md)
-* [Building on Linux](BuildInstructionsOther.md)
+
+-   [Build Instructions](BuildInstructionsLadybird.md)
```

Comment:
Changing all of these from `*` to `-` doesn't seem necessary? Especially with 2 spaces after each `-`. Makes it harder  to see what actually changed.